src/main/java/uk/gov/hmcts/reform/translate/helper/DictionaryUtils.java

```diff
@@ -0,0 +1,29 @@
+package uk.gov.hmcts.reform.translate.helper;
+
+import org.apache.commons.lang3.StringUtils;
+import uk.gov.hmcts.reform.translate.model.Dictionary;
+
+import java.util.Map;
+import java.util.function.Predicate;
+
+
+public final class DictionaryUtils {
+
+    private static final Predicate<String> isTranslationNotNull = translation -> !StringUtils.isEmpty(translation);
+
+    public static boolean isTranslationBodyEmpty(final Dictionary dictionaryRequest) {
+        return dictionaryRequest.getTranslations() == null || dictionaryRequest.getTranslations().isEmpty();
+    }
+
+    public static boolean hasAnyTranslations(final Dictionary dictionaryRequest) {
+        return dictionaryRequest.getTranslations().values().stream().anyMatch(isTranslationNotNull);
```

Comment:
Could the translations be null here ?

Reply:
Not at this point. there are other validations that takes place before this one. dictionaryRequest.getTranslations() is validated to not be null